modules/microshift-uploading-images-to-mirror.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deploy-with-mirror-registry.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-uploading-container-images-to-mirror_{context}"]
+= Uploading container images to a mirror registry
+
+To use your container images at an air-gapped site, upload them to the mirror registry using the following procedure.
+
+.Prerequisites
+
+* You are logged into a host with access to `microshift-quay`.
+* The `.pull-secret-mirror.json` file is available locally.
+* The `microshift-containers` directory contents are available locally.
+* If you use self-signed certificates, you have specified a Subject Alternative Name in the certificates.
```

Comment:
Where is this coming from?

Reply:
https://docs.google.com/document/d/1aqgjxP2YqIGi2djU96vN2yykDZpIC8s2hg4QHTCz390/edit#heading=h.om7xznfkk9bg

Reply:
The prereqs are from the beginning of the document. We can adjust as needed.

Reply:
No, it isn't. That was one I wasn't sure we needed. I saw it here when I followed the links we are giving for more information: https://docs.openshift.com/container-platform/4.13/installing/disconnected_install/installing-mirroring-installation-images.html#installation-mirror-repository_installing-mirroring-installation-images 
Can we delete this prereq for SANs here, then?